apps/site/lib/site/transit_near_me.ex

```diff
@@ -125,6 +125,15 @@ defmodule Site.TransitNearMe do
       |> expand_route_id()
       |> schedules_fn.(direction_id: direction_id, date: date)
 
+    # if there are any schedules without a trip, we need to ignore the cache
+    # because the trip ids in the cache has probably been changed during a deploy
+    if Enum.any?(schedule_data, fn sched -> Map.get(sched, :trip) === nil end) do
```

Comment:
@thecristen this is the issue with the deploy. If the provided date is outside the current GTFS file, `schedule_data` is an `{:error, _}` tuple, which is not an Enumerable.

Reply:
💡 💡 💡 💡 💡 💡 💡